sleap/gui/app.py

```diff
@@ -1155,6 +1155,7 @@ def _update_gui_state(self):
             "generate_button"
         ].setEnabled(has_videos)
         self._buttons["remove session"].setEnabled(has_selected_session)
+        self._buttons["link video"].setEnabled(has_selected_video)
```

Comment:
We require three items to perform the `LinkVideoToSession` command: 
- a selected `RecordingSession` 
- a selected `Video` and a 
- selected `Camcorder`
We should probably check that `has_selected_video and has_selected_session and has_selected_camera`.
```suggestion
        self._buttons["link video"].setEnabled(has_selected_video and has_selected_session and has_selected_camera)
```